supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -159,7 +159,7 @@ The practice of having both modes together is often referred to as _hybrid_, _ag
 
 [[binary-rpm]]
 ==== image:images/yes.png[yes] binary RPM (noun)
-*Description*: A _binary RPM_ is an RPM package that contains the binaries built from sources and patches.
+*Description*: A _binary RPM_ file is an RPM package that contains the binaries built from sources and patches.
```

Comment:
Should the actual name of the entry now include "file"? Or "package" instead of "file"?

Reply:
Yeah, I debated this one at the time in my head, and I agree that the title should match the description entry use. From my readings, an RPM package contains files such as the binary RPM file. I do seem some mixed use of the abbreviation BRPM, but it does not look that common so I'm unsure if I should mention BRPM.